### ISSUE TITLE
USWDS-Site - Enhancement: Add missing accessibility test elements to main component pages

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds-site#2593]
## Typography accessibility guidance button

Just a heads up, it looks like the typography component page is missing the accessibility guidance callout and link found on other pages.

![image](https://github.com/uswds/uswds-site/assets/25211650/5fccbc99-04ca-4b9d-9d01-1a770e60a22a) 

I believe this is because the typography page front matter is missing a components attribute since it isn't a component, and that's what the include uses to include the section:

```md
<!-- component-guidance.html -->
{% assign componentName = page.component.name | default: page.title | downcase %}
{% assign componentKey = componentName | slugify %}
{% assign settingsKey = componentKey %}
{% assign checklistData = site.data.accessibility-tests[componentKey] %}

...

{% if checklistData %}
{% include accessibility-tests/a11y-note.html %}
{% endif %}
```

Would we want to address this?

CC: @amyleadem  @mejiaj

_Originally posted by @mahoneycm in https://github.com/uswds/uswds-site/issues/2502#issuecomment-2027671646_
